examples/logging/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"github.com/hashgraph/hedera-sdk-go/v2"
+	"os"
+)
+
+func main() {
+	var client *hedera.Client
+	var err error
+
+	// Retrieving network type from environment variable HEDERA_NETWORK
+	client, err = hedera.ClientForName(os.Getenv("HEDERA_NETWORK"))
+	if err != nil {
+		println(err.Error(), ": error creating client")
+		return
+	}
+
+	// The client comes with default logger.
+	// We can also set a custom logger for this clien.
```

Comment:
typo. but w/e we can fix with the PR that would be adding tests.

Reply:
Yea, thanks